modules/hc_resource/hc_resources.module

```diff
@@ -33,7 +33,96 @@ function hc_resource_help($route_name, RouteMatchInterface $route_match)
 /**
  * Updates changing format of contact address.
  */
-function hc_resource_update_8001()
+/**
+ * Change format of contact address to a text_long.
+ */
+function hc_resources_update_8001()
 {
-  \Drupal::service('config.installer')->installDefaultConfig('module', 'hc_resource');
+  $fields = [
```

Comment:
I'm not 100% sure why you are adding this as an array, then doing a for loop for one field, but I suppose it works.

Reply:
I got this from a sample code site, but then I figured that we may want to keep it just in case we wanted to do more than one.

Reply:
ok got it.